CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+## Fleet 4.50.0 (May 22, 2024)
+
+### Endpoint Operations
+
+- Added integration so Fleet server can communicate with https://fleetdm.com to get AI generated policy descriptions and resolutions.
+- Added flag to enable deletion of old activities and associated data in cleanup cron job.
+- Added support for escaping `$` (with `\`) in gitops yaml files.
+- Added a new command `fleetctl api` to easily use fleetctl to hit any REST endpoint via the CLI.
```

Comment:
While this was done by MDM I will admit it sounds like an endpoint ops feature so I leave it up to you to move or keep. Not sure where that line is